.github/workflows/kv-vault.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       - name: Build native executable
         run: |
           chmod +x ~/.cargo/bin/cross
-          cross build --release --target ${{ matrix.target }}
+          XDG_CACHE_HOME=${HOME}/.cache cross build --release --target ${{ matrix.target }}
```

Comment:
Is this directory guaranteed to exist on the runner, or do we need a `mkdir` to be safe?

Reply:
Good call, I can add a mkdir to be safe